tusclient/fingerprint/fingerprint.py

```diff
@@ -0,0 +1,33 @@
+"""
+An implementation of of <tusclient.figerprint.interface.Figerprint>,
+using the hashlib to generate an md5 hash based on the file content
+"""
+import hashlib
+
+from . import interface
+
+
+class Fingerprint(interface.Fingerprint):
+    BLOCK_SIZE = 65536
+
+    def get_fingerprint(self, fs):
+        """
+        Return a unique fingerprint string value based on the file stream recevied
+        :Args:
+            - fs[file]: The file stream instance of the file for which a fingerprint would be generated.
+        :Returns: fingerprint[str]
+        """
+        hasher = hashlib.md5()
+        # we encode the content to avoid python 3 uncicode errors
+        buf = self._encode_data(fs.read(self.BLOCK_SIZE))
+        while len(buf) > 0:
+            hasher.update(buf)
+            buf = fs.read(self.BLOCK_SIZE)
+        return hasher.hexdigest()
```

Comment:
Would it make sense to return `"md5:" + haser.hexdigest()` to identify that this fingerprint is from a MD5?